servatrice/src/serversocketinterface.cpp

```diff
@@ -1794,3 +1792,9 @@ void WebsocketServerSocketInterface::binaryMessageReceived(const QByteArray &mes
 
     processCommandContainer(newCommandContainer);
 }
+
+// TODO make this configurable?
+bool isPasswordLongEnough(const int passwordLength) const
```

Comment:
Nice refactor, but this method implementation is missing the class name:
```
bool AbstractServerSocketInterface::isPasswordLongEnough(.....
```

Reply:
Indeed, my bad. Fixed